game/mainwindow.cpp

```diff
@@ -687,16 +704,32 @@ void MainWindow::onMarvinKey() {
       setFullscreen(!SystemApi::isFullscreen(hwnd()));
       break;
     case Event::K_F4:
+      if(Gothic::inst().isMarvinEnabled()) {
+        if(auto camera = Gothic::inst().camera())
+          camera->reset();
+        }
       break;
     case Event::K_F5:
-      Gothic::inst().quickSave();
-      // note: camera lock
+      if(Gothic::inst().isMarvinEnabled()) {
+        if(auto camera = Gothic::inst().camera()) {
+          camera->setMarvinMode(Camera::M_Freeze);
+          }
+        } else {
+        Gothic::inst().quickSave();
```

Comment:
While this is correct, not having `F5` in debug play would be painfully annoying. Maybe remove `camera-freeze`, from debug build?